[2021-06-27 10:12:36,260] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: docker_dag.docker_command 2021-06-27T10:00:00+00:00 [queued]>
[2021-06-27 10:12:36,327] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: docker_dag.docker_command 2021-06-27T10:00:00+00:00 [queued]>
[2021-06-27 10:12:36,329] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 10:12:36,331] {taskinstance.py:1068} INFO - Starting attempt 2 of 2
[2021-06-27 10:12:36,339] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-27 10:12:36,359] {taskinstance.py:1087} INFO - Executing <Task(DockerOperator): docker_command> on 2021-06-27T10:00:00+00:00
[2021-06-27 10:12:36,365] {standard_task_runner.py:52} INFO - Started process 497 to run task
[2021-06-27 10:12:36,369] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'docker_dag', 'docker_command', '2021-06-27T10:00:00+00:00', '--job-id', '235', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dag_docker_teste.py', '--cfg-path', '/tmp/tmprsd98kl4', '--error-file', '/tmp/tmp31apbyjc']
[2021-06-27 10:12:36,370] {standard_task_runner.py:77} INFO - Job 235: Subtask docker_command
